cypress/integration/features/add-action.spec.js

```diff
@@ -66,6 +56,7 @@ context('Add action form', () => {
       );
 
       cy.get('[data-testid=add-action-button-test]').click();
+      cy.get('[data-testid=add-action-button-test]').should('not.exist');
```

Comment:
Waits for the button to disappear, since this is hidden after the API call has completed.